charts/rstudio-pm/values.yaml

```diff
@@ -100,6 +100,18 @@ pod:
   serviceAccountName: false
   # -- annotations is a map of keys / values that will be added as annotations to the pods
   annotations: {}
+  # -- Additional labels to add to the rstudio-pm pods
+  labels: {}
+  # -- An array of containers that will be run alongside the main pod
+  sidecar: false
```

Comment:
What is the purpose of specifying "sidecar" containers when we already support `extraContainers`?